presto-main/src/test/java/io/prestosql/security/TestAccessControlConfig.java

```diff
@@ -26,18 +25,18 @@
     public void testDefaults()
     {
         ConfigAssertions.assertRecordedDefaults(ConfigAssertions.recordDefaults(AccessControlConfig.class)
-                .setAccessControlFile(null));
+                .setAccessControlFiles(""));
     }
 
     @Test
     public void testExplicitPropertyMappings()
     {
         Map<String, String> properties = new ImmutableMap.Builder<String, String>()
-                .put("access-control", "someFile")
+                .put("access-control", "a,b,c")
                 .build();
 
         AccessControlConfig expected = new AccessControlConfig()
-                .setAccessControlFile(new File("someFile"));
+                .setAccessControlFiles("a,b,c");
```

Comment:
The expected should use `File`s (fully decoded value), like in https://github.com/prestosql/presto/blob/a98a3b91b9c094d32d0268615ea11505cf3fd614/presto-hive/src/test/java/io/prestosql/plugin/hive/TestHdfsConfig.java#L67